product_docs/docs/edbcloud/beta/administering_cluster/organization_and_access/02_users.mdx

```diff
@@ -1,24 +1,91 @@
 ---
-title: "Users"
+title: "Managing Users and Access"
 ---
 
-## Viewing Users
-If you have a role that grants you  access to Admin screen, you can view all users from your organization that has been logged in at least once.
+!!!
+Access to the Admin screen is required for these actions
 
-To view users:
-- Navigate to Admin > Users.
 
-The list of users is sorted by the time of the latest login.
+# Roles
+  
+## Default Roles
+Access to EDB Cloud UI is based on permissions. Roles are sets of permissions. Roles are a convenient abstraction to manage permissions assigned to users.
```

Comment:
- I would remove "UI". 
- I would edit the 3rd sentence to: 'Roles are used to manage permissions assigned to users.'

Reply:
Updataed